emailproxy.py

```diff
@@ -324,8 +386,44 @@ def add_account(username):
     @staticmethod
     def save():
         if AppConfig._LOADED:
+            def store_and_clear_aws_secrets():
+                TOKEN_KEYS = ['token_salt','access_token','access_token_expiry','refresh_token']
+
+                # Create deep copy of config (to write tokens to AWS Secrets Manager, not to local config file)
+                appconfig_to_save = configparser.ConfigParser()
+                appconfig_to_save.read_dict(AppConfig._PARSER)
+
+                # Create dict of AWS Secret IDs across all accounts
+                aws_secrets = dict.fromkeys([appconfig_to_save[account]['aws_secret'] for account in accounts_using_aws_secret], {})
+
+                # Populate dict with OAuth tokens from each account
+                for account in accounts_using_aws_secret:
+                    aws_secrets[appconfig_to_save[account]['aws_secret']][account] = { key : appconfig_to_save[account][key] for key in TOKEN_KEYS }
+                    for key in TOKEN_KEYS:
+                        appconfig_to_save.remove_option(account, key)
+
+                # Update AWS Secrets
+                aws_client = boto3.client('secretsmanager')
```

Comment:
```suggestion
                try:
                    aws_client = boto3.client('secretsmanager')
                except botocore.exceptions.BotoCoreError:
                    Log.error('Unable to load AWS Secrets Manager - please make sure you have set up the AWS SDK for Python')
                    return
```

Reply:
@simonrob I think your proposal here is related to your feedback regarding line 294:

> I'm a little reluctant to exit the proxy when errors or misconfigurations occur (both here and other places) unless they are absolutely critical. Perhaps just return here without loading?

In which case I'll update this code chunk based on the resolution to our conversation in that thread.

If I'm misinterpreting your intention here, let me know!